SECURITY.md

```diff
@@ -16,12 +16,12 @@ Our most critical targets are:
 
 * WooCommerce core (this repository)
 * WooCommerce [Blocks](https://wordpress.org/plugins/woo-gutenberg-products-block/) and [Admin](https://wordpress.org/plugins/woocommerce-admin/) packages and plugins
-* WooCommerce.com -- the primary marketplace and marketing site, and all of it subdomains, e.g. [developer.woocommerce.com](https://developer.woocommerce.com/)
+* woo.com -- the primary marketplace and marketing site, and all of it subdomains, e.g. [developer.woocommerce.com](https://developer.woo.com/)
```

Comment:
@coreymckrill Migration to developer.woo.com is in progress, but it will be finished really soon. Surely before the release of this code change.

Reply:
I explicitly avoided that sub-domain because it wasn't yet working -- this one just slipped through. I'll make sure to replace any other occurrences.